Dockerfile

```diff
@@ -1,5 +1,13 @@
-FROM circleci/openjdk:9
-RUN sudo apt-get update && sudo apt-get install gettext docker python-pip python-setuptools wget curl apt-transport-https
+FROM circleci/openjdk:11
```

Comment:
I assume our production Corretto images will be based on Amazon Linux? Wouldn't it make sense to also use them in the CI pipeline and then install everything required for CircleCI instead of using the Debian based CircleCI OpenJDK images and install Corretto?

Reply:
@tkaschwig 
Yes, when creating the production Docker image for our services we will use this Docker image: https://dev2.signavio.com/signavio/docker-common/blob/master/amazon-corretto-11/Dockerfile
For pex-backend you can see it here in this PR: https://github.com/signavio/pex-backend/pull/11/files#diff-3254677a7917c6c01f55212f86c57fbfR1

Regarding CircleCI pipeline I am not sure what exactly is required and how to set them up. So I found it easier to use one of the images provided by CircleCI, in this case the closest with OpenJDK, and then install Corretto and change default JVM to Corretto.

Reply:
Have you tried to just use the plain Amazon Corretto images in CircleCI?

Reply:
I don't think that would work.

Reply:
You don't think or you have tried out? ;-)

Reply:
Ok, will quickly test it.